Whiskey/Whiskey.psd1

```diff
@@ -159,7 +159,10 @@
 
             # ReleaseNotes of this module
             ReleaseNotes = @'
-* Fixed: Whiskey's error view displays too much information sometimes.
+* The `GetPowerShellModule` task now supports installing prerelease versions of modules. Set the `AllowPrerelease` property to `true`.

```

Comment:
This release note format still gets me because the cumulative change set does not carry forward :|

Reply:
We should make a task that makes a cumulative set of release notes.